src/memory/vaddr.c

```diff
@@ -114,7 +114,11 @@ static inline word_t vaddr_read_internal(void *s, vaddr_t addr, int len, int typ
     isa_misalign_data_addr_check(addr, len, type);
   }
 #endif
+#ifdef CONFIG_RVV
+  if (unlikely(mmu_mode == MMU_DYNAMIC || (mmu_mode == MMU_TRANSLATE && ((struct Decode*)s)->v_is_vx == 0) )) {
```

Comment:
what is v_is_vx?

Reply:
The addresses requested for address translation in the vectro instruction are aligned according to the width of the element, and the translation is required when performing the check,Since the Vectore independent check is not implemented, it needs to be skipped